app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] As mentioned in the project requirements, when users sign up, they should be prompted to confirm their emails. Please add `:confirmable` to this list.

Reply:
Thanks for your advice! I will add it in the next commit